frontend/public/components/deployment.jsx

```diff
@@ -61,6 +83,11 @@ const DeploymentDetails = ({obj: deployment}) => {
   return <React.Fragment>
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Overview" />
+      {deployment.spec.paused &&
+      <p className="alert alert-info">
```

Comment:
I'd probably make this a reusable component, too, instead of repeating it. Maybe `WorkloadPausedAlert`.